content/en/docs/use-cases/confidential-ai.md

```diff
@@ -9,4 +9,74 @@ tags:
 weight: 60
 ---
  
-Coming soon
+## Federated Learning
+
+![Federated Learning](/img/FederatedLearning.png)
+
+### Attestations
+- Federated Learning requests multi-SDK attestation
+- FL Servers needs to verify all client’s trustworthiness
+- Attestation at different points, self and cross verifications via attestation service
+
+### CC Policies:
+- Bootup policy – provided by hardware vendor
+- Self-verification CC policy – user defined
+- Cross-verification CC policy  -- user defined
+
+### Protected Assets
+- **Code**: Training code (client), aggregation code (server)
+- **Data**: input data
+- **Model**:  initial model, intermediate model, output model
+- **Workspace**: check points, logs, temp data, output directory
+
+### Key Broker Service
+- Key management depends on user case, global model ownership, key release management process
+
+### Bootstrap:
+- Need a process to generate the keys, policies and input them into key-vault to avoid tempering
+
+### Concerns when using federated learning
+- Trustworthy of the participants
+- Execution code tampering
+- Model tampering
+- Model inversion attack
+- Model theft during training
+- Private data leak 
+
+### CoCo will not protect if
+- code is already flawed at rest
+- data is already poisoned at rest
+- model is already poisoned at rest
+
+## Multi-Party Computing
```

Comment:
Multi-party computing could be more generic than AI. Do we want it here or in another use case?